hummingbot/market/bittrex/bittrex_in_flight_order.pyx

```diff
@@ -0,0 +1,67 @@
+from decimal import Decimal
+from typing import Optional, Dict, Any
+
+from hummingbot.core.event.events import OrderType, TradeType
+from hummingbot.market.bittrex.bittrex_market import BittrexMarket
+from hummingbot.market.in_flight_order_base import InFlightOrderBase
+
+
+cdef class BittrexInFlightOrder(InFlightOrderBase):
+    def __init__(self,
+                 client_order_id: str,
+                 exchange_order_id: Optional[str],
+                 symbol: str,
+                 order_type: OrderType,
+                 trade_type: TradeType,
+                 price: Decimal,
+                 amount: Decimal,
+                 initial_state: str = "OPEN"):
+        super().__init__(
+            BittrexMarket,
+            client_order_id,
+            exchange_order_id,
+            symbol,
+            order_type,
+            trade_type,
+            price,
+            amount,
+            initial_state
+        )
+
+    @property
+    def is_done(self) -> bool:
+        return self.last_state in {"CLOSED"}
+
+    @property
+    def is_failure(self) -> bool:
+        return NotImplementedError
+
+    @property
+    def is_cancelled(self) -> bool:
+        return self.last_state in {"CLOSED"}
```

Comment:
This looks concerning... as it means we can't distinguish between cancelled and completed.